src/ray/util/throttler_test.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtest/gtest.h"
+
+#include <cstdlib>
+#include <thread>
+
+#include "ray/util/throttler.h"
+
+TEST(ThrottlerTest, BasicTest) {
```

Comment:
This means the test takes 20 seconds right? Can we just use the logical timer like `PullManager`'s `get_time`?

Reply:
+1

Reply:
Let me check it